magefile.go

```diff
@@ -160,6 +160,12 @@ func buildCaddy(goos string) error {
 	return sh.RunWithV(env, "xcaddy", buildArgs...)
 }
 
+// BuildExample builds the example deployment. Requires docker-compose.
+func BuildExample() error {
+	mg.SerialDeps(BuildCaddyLinux)
```

Comment:
BuildExample is meant to build the caddy binary to be run in docker hence linux.

Reply:
I see. I was trying to make these steps work:
```
# in terminal 2
go run mage.go buildExample
./build/caddy run --config example/Caddyfile --adapter caddyfile
```
after `buildExample`, `./build/caddy` would not exists.

The right fix might be to just call `go run mage.go buildCaddy` instead of `go run mage.go buildExample`

Reply:
Got your point!

Reply:
Got your point, actually I shouldn't have used buildExample but buildCaddy instead. Thanks for the heads up!